tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -168,8 +173,10 @@ fi
 
 if policy_tests_enabled; then
 	ensure_yq
+	export use_containerd_pull=0	
```

Comment:
Thoughts:
1. Rename this variable to GENPOLICY_CONTAINERD_PULL.
2. If GENPOLICY_CONTAINERD_PULL was not defined at the beginning of this script, then use the default values you are using here.
3. If GENPOLICY_CONTAINERD_PULL was defined at the beginning of this script:
- Honor that value - by running all of the tests using it.
- If GENPOLICY_CONTAINERD_PULL was `no` then skip the new "(default image pull)" test.

Even better, use GENPOLICY_PULL_METHOD="oci-distribution" and GENPOLICY_PULL_METHOD="containerd" instead of GENPOLICY_CONTAINERD_PULL="no" and GENPOLICY_CONTAINERD_PULL="yes".

@sprt , do you have better ideas here?